testgrid/cmd/configurator/prow_test.go

```diff
@@ -632,6 +634,216 @@ func Test_applySingleProwjobAnnotation_WithDefaults(t *testing.T) {
 
 }
 
+func Test_applySingleProwjobAnnotations_AutomaticFieldPopulation(t *testing.T) {
```

Comment:
Seems like you're only interested in verifying  `OpenTestTemplate` here, I would consider modifying your test case to only check that field, or folding into test cases above that are checking all fields
```suggestion
func Test_applySingleProwjobAnnotations_OpenTestTemplate(t *testing.T) {
```

Reply:
blocking nit: I would like to see a test case confirming this overwrites test grid's default OpenTestTemplate

Reply:
> Non-blocking nit: I would consider modifying your test case to only check that field, or folding into test cases above that are checking all fields

makes total sense, I've gone for the second option.

Reply:
> blocking nit: I would like to see a test case confirming this overwrites test grid's default OpenTestTemplate

Done, added as part of Test_applySingleProwjobAnnotations_OpenTestTemplate